test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -113,6 +112,7 @@ public class CorfuReplicationClusterConfigIT extends AbstractIT {
 
     @Before
     public void setUp() throws Exception {
+        super.setUp();
```

Comment:
why do we need the extra cleanup and setup from the base class?

Reply:
Thanks, Done. (@Jas0nch had added it)